third_party/proton/test/test_override_helper.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
This is indeed a test file since it contains `test_*` functions. The previous help should be a helper file

Reply:
updated